network/sites.csv

```diff
@@ -959,8 +959,10 @@ GR4A,10,-41.886098794,173.593746503,609,,WGS84,Unknown,2001-07-30T22:00:00Z,2002
 GR5A,10,-41.892998957,173.593346468,525,,WGS84,Unknown,2001-07-29T00:00:01Z,2002-01-22T04:00:00Z
 GRAM,10,-41.197980696,172.779639286,460,,WGS84,Unknown,1993-10-24T08:36:00Z,1994-03-01T16:34:00Z
 GRAS,20,-38.664511,178.025314,26,,WGS84,Unknown,2021-05-13T01:08:00Z,9999-01-01T00:00:00Z
-GRAS,21,-38.664264,178.025344,26,,WGS84,Unknown,2021-05-13T01:08:00Z,9999-01-01T00:00:00Z
-GRAS,22,-38.664061,178.025328,26,,WGS84,Unknown,2021-05-13T01:08:00Z,9999-01-01T00:00:00Z
+GRAS,21,-38.664355,178.02503326,26,,WGS84,Topographic Map,2021-05-13T01:08:00Z,2023-11-25T04:46:00Z
```

Comment:
latitude for 21 is `178.025033`?

Reply:
Fixed now sorry, looks like I accidentally deleted the comma between longitude and elevation for that extra 26 to get there.

Reply:
no worries, thanks @Thomas-Benson 👍